styles/themes/charcoal.less

```diff
@@ -0,0 +1,226 @@
+/* ------------------------------------------------------------------------ */
+/* Theme coloring for Charcoal */
+
+@import "../layout.less";
+
+@theme-name: "charcoal";
+
+@page-background: #262626;
+@page-text: #e2e2e2;
+@header-background: #1a1a1a;
+@header-text: #d5d5d5;
+@navbar-background: black;
+@navbar-text: #cdcdcd;
+@sidebar-background: #1a1a1a;
+@sidebar-text: #d5d5d5;
+@footer-background: black;
+@footer-text: #cdcdcd;
+@heading-color: #bcbcbc;
+@link-color: #b3ccff; // #6699ff;
+@visited-color: #dfb9b9; // #f5bcf5; // #ffcce3; // #dfb9b9; // #bb7070;
+@active-color: #ff66aa;
+@sans-serif-font: Verdana, Helvetica, sans-serif;
+
+/* invert() for dark theme. none or opacity() for light
+ * currently disabled, as it's primarily meant for the
+ * proofreading interface images, and that's not themed
+ * yet
+ */
+// @image-property: invert(85%);
+
+/* Form and menu colors */
+@form-text-background: #343434;
+@form-text-color: @page-text;
+
+// Declared here so that the light themes can use browser default
+// input and select styling
+input,
+button,
+select {
+    background-color: @form-text-background;
+    color: @form-text-color;
+    .solid-border;
+}
+
+/* ------------------------------------------------------------------------ */
+/* Common color definitions
+ * Where a particular color family has multiple shades available, a
+ * suffix indicates relative contrast as compared with the default
+ * @page-background color.
+ * All of the color names defined in this file are the default colors
+ * for themes with a white page-background with black page-text, but
+ * for themes that have other than a white page-background with black
+ * page-text, the colors can be overridden in that theme's .less file
+ */
+
+/* Shades based on @page-background. Various contrast levels, commonly
+ * used for table rows or other containers that should stand out from
+ * the page background
+ */
+@table-cell-base-highc: #121212; // black;
+@table-cell-base-mediumc: #3a3a3a; //#121212;
+@table-cell-base-lowc: #4d4d4d; //#454545;
+
+/* Background colors with various meanings, some needing two different
+ * contrast levels
+ */
+@table-cell-ok-highc: #004d00;     // high contrast, green family
+@table-cell-ok-lowc: #009900;      // low contrast, green family
+@table-cell-alert: #b34700;        // yellow/orange family
+@table-cell-not-ok-highc: #66001a; // high contrast, red family
+@table-cell-not-ok-lowc: #b3002d;  // low contrast, red family
+@table-cell-pending: #b35900; //#e67300;      // orange family -- contrast iffy
+
+/* Special text colors that have a particular meaning that are not
+ * related to the theme colors but nonetheless may need to be
+ * adjusted depending on the theme. Also, needed variation on
+ * theme default page-text color
+ */
+@text-error: #ff8080;        // red family    -- too bright
```

Comment:
Is this still too bright and the one two down too dark, or were these left over from your development?

Reply:
I'm not sure the colors are perfect, but the colors have been tweaked so that they're no longer too bright/dark.